sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -280,6 +280,9 @@
    */
   private @NotNull RequestSize maxRequestBodySize = RequestSize.NONE;
 
+  /** Controls if the `tracestate` header is attached to envelopes and HTTP client integrations. */
+  private boolean traceSampling;
```

Comment:
is it disabled by default?
if this is intended for mobile too, let's add the ability to turn it on via Android Manifest
See: ManifestMetadataReader

Reply:
👍